app/src/main/java/com/theuhooi/uhooipicbook/modules/monsterlist/MonsterListFragment.kt

```diff
@@ -59,6 +63,21 @@ class MonsterListFragment : Fragment() {
         this.listener = null
     }
 
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        super.onCreateOptionsMenu(menu, inflater)
+        inflater.inflate(R.menu.menu_monster_list, menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            R.id.action_nav_licenses -> {
+                findNavController().navigate(MonsterListFragmentDirections.actionListToLicenses())
+                true
+            }
+            else -> false
```

Comment:
公式ドキュメントによるとデフォルトで `false` を返すので、 `else` の場合は常に `false` を返しても問題なさそうですが、公式ドキュメントの通りにオーバーライド前のメソッドを呼び出すのがよさそうです。
ref: https://developer.android.com/guide/topics/ui/menus?hl=ja#RespondingOptionsMenu

```suggestion
            else -> super.onOptionsItemSelected(item)
```